core/src/main/java/org/apache/iceberg/ManifestGroup.java

```diff
@@ -163,20 +171,26 @@ public CloseableIterable<FileScanTask> planFiles() {
     });
 
     DeleteFileIndex deleteFiles = deleteIndexBuilder.build();
-
     boolean dropStats = ManifestReader.dropStats(dataFilter, columns);
     if (!deleteFiles.isEmpty()) {
       select(ManifestReader.withStatsColumns(columns));
     }
 
     Iterable<CloseableIterable<FileScanTask>> tasks = entries((manifest, entries) -> {
+      scanMetrics.addedDataFiles().increment(manifest.addedFilesCount());
+      scanMetrics.deletedDataFiles().increment(manifest.deletedFilesCount());
```

Comment:
I think I see what you're doing with these counters now. These end up tracking the total number of data or delete files in manifests that are scanned, right?

Reply:
yes correct. The naming is probably bad in `ScanMetrics`, so I'm completely open to suggestions on naming and also on what metrics to track and where to retrieve them from. The initial idea was that there are just **some metrics** gathered and reported